src/tasks/external_task.cpp

```diff
@@ -47,8 +49,12 @@ void external_task::sandbox_init()
 {
 #ifndef _WIN32
 	if (task_meta_->sandbox->name == "isolate") {
+		sandbox_limits limits(*limits_);
+		if (this->get_type() == task_type::INITIATION) {
+			limits.share_net = true; // initiation (compilation) tasks may use internet to download stuff
```

Comment:
A little TODO would be nice... This is probably not the final, nor the ideal place to set this